core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1630,7 +1630,7 @@ private long loadDirectoryMetadata(AlluxioURI path, LoadMetadataOptions options)
       throws IOException, InvalidPathException, AccessControlException, FileDoesNotExistException {
     if (mInodeTree.inodePathExists(path)) {
       Inode inode = mInodeTree.getInodeByPath(path);
-      if (inode.isPersisted()) {
+      if (inode.isPersisted() || path.isRoot()) {
```

Comment:
I don't think so because root is not persisted if we don't have UFS. But I think it is a matter of convention. We can change that and say root is always considered as persisted no matter what. 

But I think that is out of the scope of this PR. The other option is to remove InodeTree.java, line 284 and makes sure that function works with root.  And fix all of its callers.